sw/Modules/Comms/Inc/can_sensors.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * can_sensors.h
+ *
+ *  Created on: Mar 19, 2025
+ *      Author: Hardy Yu
+ */
+#ifndef CAN_SENSORS_H_
+#define CAN_SENSORS_H_
+
+#include <stdint.h>
+#include "canard.h"
+#include "dronecan_msgs.h"
+#include "imu.h"
```

Comment:
Nit: I prefer having C standard library, auto-generated/3rd-party library, and our own library includes separated.

For ex: `asm330lhh.c`
<img width="268" alt="image" src="https://github.com/user-attachments/assets/a652fb21-4588-41be-b6a8-72e69d026f08" />